README.md

```diff
@@ -5,7 +5,7 @@ Template project for Java 8 web app stack using spark, redis, bootstrap, jade an
 
 1. clone me
 2. install and start redis
-3. install browserify using npm -g browserify
+3. run npm install --save
```

Comment:
What does --save do here? I thought that was for installing a new package and saving to package.json?